server/channels/db/migrations/postgres/000122_preferences_value_length.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Preferences ALTER COLUMN TYPE VARCHAR(2000);
```

Comment:
Rolling this back took around 90 seconds with the same setup described in my other comment. That would start to get impactful when there's more users, but I assume that's more acceptable given we this is less likely to be done. If they get to the point where this takes more than a few minutes, they probably should've done a DB backup before attempting to upgrade to begin with 😛